README.md

```diff
@@ -46,12 +46,17 @@ You can configure the bundle in your `config.yml`. Full Example:
 webfactory_piwik:
     # Required, no default. Must be set to the site id found in the Matomo control panel
     site_id: 1
-    # Required, has default. Usually, you only want to include the tracking code in a production environment
-    disabled: '%kernel.debug%'
     # Required. no default. Hostname and path to the Matomo host.
     piwik_host: my.piwik.hostname
-    # Required, has default. Path to the tracking script on the host.
+    # Optional, has default. Usually, you only want to include the tracking code in a production environment
```

Comment:
Was it necessary to change the order of these lines?

Reply:
No, but I would like to see the required keys before the optional ones, because these will be more importend.